### DISCUSSION TITLE
Auto save and Load

### DISCUSSION THREAD

**Discussion by 7bajwa** [manuelernestog/weektodo#170] [Ideas]
Hello,

Thank you very much for creating the beautifully simple app.

If we can link the data to a folder or file then auto save with changes will be great. 
I export the file to import it on other devices then bring it to the device I use.
I sorted this with common place on my one drive so I just replace it from the device I updated.
If we can make it possible to auto save of we make the changes into the lists will be great.
and it look at the file if it is older than it load that as well. I made a small eco system without the cloud authorization.

I think this will stream line the workflow.

Thanks,

**Comment by L0nelyPlut0** [comment]
I agree with this, a sync would be nice. I'd like to make both to-do lists on both my phone and PC.

**Comment by NoSpamMaam** [comment]
Nice app. Agree with the above.
